src/components/progressTracker/ProgressTracker.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { ReactNode } from 'react';
+import cx from 'classnames';
+import theme from './theme.css';
+
+import Box from '../box';
+import ProgressStep, { ProgressStepProps } from './ProgressStep';
+import { GenericComponent } from '../../@types/types';
+
+interface ProgressTrackerProps {
+  /** Whether or not all steps are completed */
+  done?: boolean;
+  /** The number of the step which is currently active */
+  currentStep: number;
+  /** The steps to display inside the progress tracker */
+  children?: ReactNode;
+  /** Color theme of the progress tracker. */
+  color: 'neutral' | 'mint' | 'aqua' | 'violet' | 'gold' | 'ruby' | 'teal';
+  /** Where to position the labels. Alternating allows for wider labels. */
+  labelPosition: 'top' | 'alternating' | 'bottom';
```

Comment:
Q: are color, currentStep and labelPosition required? They have a default value, so they can be optional?

Reply:
True! Will edit ✅